src/scancode/cli.py

```diff
@@ -458,11 +465,35 @@ def scancode(ctx,
             full_root=full_root,
             ignore=user_ignore)
 
+        # Find all scans that are both enabled and have a valid function
+        # reference. This deliberately filters out the "info" scan
+        # (which always has a "None" function reference) as there is no
+        # dedicated "infos" key in the results that "plugin_only_findings.has_findings()"
+        # could check.
+        # FIXME: we should not use positional tings tuples for v[0], v[1] that are mysterious values for now
+        active_scans = [k for k, v in scanners.items() if v[0] and v[1]]
+
+        plugins_invoked = False
```

Comment:
This is too generic as a variable name but I will merge this and rename when I push